src/DataTable/ExpandRow.jsx

```diff
@@ -17,7 +17,7 @@ ExpandRow.propTypes = {
   /** Row data that is received from `react-table` API. */
   row: PropTypes.shape({
     /** Specifies if row is in expanded state. */
-    isExpanded: PropTypes.bool.isRequired,
+    isExpanded: PropTypes.bool,
```

Comment:
`react-table` sometimes actually returns `undefined` instead of `false` which caused console errors